manifests/ca.pp

```diff
@@ -58,24 +58,4 @@
       ensure  => absent,
     }
   }
-
-  if $deploy {
-    include certs::config::deploy
-
-    file { $certs::katello_default_ca_cert:
-      ensure => file,
-      source => $default_ca_path,
-      owner  => 'root',
-      group  => 'root',
-      mode   => '0644',
-    }
-
-    file { $katello_server_ca_cert:
```

Comment:
That's a parameter on this class, but I think it's now unused and can be dropped.

While checking, I noticed 2 others were already unused: https://github.com/theforeman/puppet-certs/pull/495